### ISSUE TITLE
Problem regarding `/Manifest` after `Manifest URL`

### ISSUE THREAD

**Issue by KazumiHARADA** [google/ExoPlayer#3786]
When the last part of `Manifest URL` is `(filter=phone_hd_001)` , it seems that the library is adding `/Manifest` after the URL. Can you remove that?
```
private SsMediaSource(SsManifest manifest, Uri manifestUri,
      DataSource.Factory manifestDataSourceFactory,
      ParsingLoadable.Parser<? extends SsManifest> manifestParser,
      SsChunkSource.Factory chunkSourceFactory, int minLoadableRetryCount,
      long livePresentationDelayMs, Handler eventHandler,
      AdaptiveMediaSourceEventListener eventListener) {
    Assertions.checkState(manifest == null || !manifest.isLive);
    this.manifest = manifest;
    this.manifestUri = manifestUri == null ? null
        : Util.toLowerInvariant(manifestUri.getLastPathSegment()).equals("manifest") ? manifestUri
            : Uri.withAppendedPath(manifestUri, "Manifest");  //// <- here
    this.manifestDataSourceFactory = manifestDataSourceFactory;
    this.manifestParser = manifestParser;
    this.chunkSourceFactory = chunkSourceFactory;
    this.minLoadableRetryCount = minLoadableRetryCount;
    this.livePresentationDelayMs = livePresentationDelayMs;
    this.eventDispatcher = new EventDispatcher(eventHandler, eventListener);
    mediaPeriods = new ArrayList<>();
  }
```
library version : r2.4.3

**Comment by ojw28** [comment]
This has already been fixed (as of 2.6.0). See https://github.com/google/ExoPlayer/commit/ce7aaab3c6adc8d8b0d6723c0e59441fcca9bce2.

**Comment by KazumiHARADA** [comment]
Thank you for the reply.
We consider to update the library inside our application.
